any_parser/__init__.py

```diff
@@ -1,4 +1,6 @@
-from any_parser.base import AnyParser
+"""AnyParser module for parsing data."""
+
```

Comment:
nit: let's add a version here like https://github.com/CambioML/any-parser/blob/main/any_parser/__init__.py#L5. Also, let's update the version to 0.0.14 in pyproject.toml https://github.com/CambioML/any-parser/blob/main/pyproject.toml